### DISCUSSION TITLE
Dns saved and unavailable , get redirect to a 404

### DISCUSSION THREAD

**Discussion by SCHMAHUB** [community/community#146394] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I created my repository as github docs explains, 'nomespaladin.github.io', when in setting i changed 'saved' the DNS to 'nomessec@org' , it showed me it was unavailable but start to redirect the name of my github website to the other one ' .org' but i cant create a new repository on my name because even like that would redirect to the same 'nomessec.org'. Any help is welcome because now i'm unable to create my website.

**Comment by mecodeatlas** [comment]
Hey @nomespaladin!

Did you get a chance to check out our [Troubleshooting forum](https://github.com/orgs/community/discussions/61073) for 404 errors and review our [documentation](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/troubleshooting-custom-domains-and-github-pages)? If you've already followed the suggested steps and guidelines, we recommend taking a look at [this discussion](https://github.com/orgs/community/discussions/23525), as it might be helpful in resolving the issue :)

**Reply by SCHMAHUB** [reply]
Issue is resolved. Thanks for the support.
